charts/cloudcasa/cloudcasa/1.0.0/Chart.yaml

```diff
@@ -1,9 +1,10 @@
 annotations:
   catalog.cattle.io/certified: partner
+  catalog.cattle.io/display-name: CloudCasa
```

Comment:
This third commit contains the actual changes that were introducing by upstream OOB by modifying the release assets we were pointing to.